Common/DataModel/TrackSelectionTables.h

```diff
@@ -16,6 +16,12 @@
 
 namespace o2::aod
 {
+namespace fwdtrack
+{
+DECLARE_SOA_COLUMN(FwdDcaX, fwddcaX, float); //! Impact parameter in X of forward track to the primary vertex
```

Comment:
```suggestion
DECLARE_SOA_COLUMN(FwdDcaX, fwdDcaX, float); //! Impact parameter in X of forward track to the primary vertex
```
(Usually this is the schema we follow, we only change the capitalization of the first character